app/views/errors.py

```diff
@@ -95,14 +96,21 @@ def get_tx_id():
     return tx_id
 
 
+def get_account_url():
+    account_url = current_app.config['RESPONDENT_ACCOUNT_URL']
+    if ACCOUNT_URL in cookie_session:
+        account_url = cookie_session[ACCOUNT_URL]
+    return account_url
+
+
 def render_template(template_name):
     tx_id = get_tx_id()
     user_agent = user_agent_parser.Parse(request.headers.get('User-Agent', ''))
 
-    return render_theme_template(session.get('theme', 'default'),
+    return render_theme_template(cookie_session.get('theme', 'default'),
```

Comment:
It looks like there's no test coverage for the `respondent_account_url` getting set by the JWT payload.

Reply:
@nealedj i have added test coverage